src/api/config/speedsettings.rs

```diff
@@ -92,7 +92,7 @@ impl Default for SpeedSettings {
         reduced_tx_set: false,
         // TX domain distortion is always faster, with no significant quality change,
         // although it will be ignored when Tune == Psychovisual.
-        tx_domain_distortion: true,
+        tx_domain_distortion: false,
```

Comment:
Is this disabled for testing, or has it become worse as a default? (If the latter, we should update or remove the above comment.)

Reply:
This is primarily for testing. I think both are true at the moment because temporal RDO is disabled when transform-domain distortion is enabled. The quality change will determined by the difference between static and dynamic temporal models. Note that the static model is frame-level.